docs/editor/intellisense.md

```diff
@@ -0,0 +1,162 @@
+| ![blue cuboid for a variable](images/intellisense/variable_icon.png) | Variable | 
+| ![blue circles connected by a blue line for an object](images/intellisense/object_icon.png) | Object |
+| ![a square with letters abc word completion](images/intellisense/word_completion_icon.png) | Word Completion |
+| ![a square with a small fold in the top left corner indicates string completion](images/intellisense/string_completion_icon.png) | String completion |
+| ![a square with ellipses forming the bottom show snippet prefix](images/intellisense/word_completion_icon.png) | Snippet Prefix |
+| ![a series of lines arranged horrizontally show a language keyword](images/intellisense/string_completion_icon.png) | Language Keyword |
+
+## Customize IntelliSense
+
+You can customize your IntelliSense experience in settings and key bindings. 
+
+### Settings
+
+The settings shown below are the default settings. You can change these settings in your settings.json file as described [here](/docs/customization/userandworkspace.md). 
+
+```js
+{
+
+    // Controls if quick suggestions should show up while typing
+    "editor.quickSuggestions": true,
+
+    // Controls if suggestions should be accepted with "Enter" - in addition to "Tab". Helps to avoid ambiguity between inserting new lines and accepting suggestions. 
+    "editor.acceptSuggestionOnEnter": true,
+
+    // Controls the delay in ms after which quick suggestions will show up. 
+    "editor.quickSuggestionsDelay": 10
+}
+```
+
+### Key Bindings
+
+The key bindings shown below are the default key bindings. You can change these key bindings in your `keybindings.json` file as described [here](/docs/customization/keybindings.md). 
+
+> **Note:** The key bindings below are the most popular key bindings for IntelliSense. There are many more key bindings relating to IntelliSense. Simply open key bindings and search for "suggest." 
+
+```js
+{
+
+   { 
+       "key": "ctrl+space",            
+       "command": "editor.action.triggerSuggest",
+        "when": "editorHasCompletionItemProvider && editorTextFocus && !editorReadonly" 
+    },
+    { 
+        "key": "tab",                   
+        "command": "acceptQuickFixSuggestion",
+        "when": "editorFocus && quickFixWidgetVisible" 
+    },
+    { 
+        "key": "enter",                 
+        "command": "acceptQuickFixSuggestion",
+        "when": "editorFocus && quickFixWidgetVisible" 
+    },
+}
+```
+
+
+## Troubleshooting IntelliSense
+
+If you find IntelliSense has stopped working, the language service may have crashed. Simply restart VS Code and this should solve the issue. If you continue to lack IntelliSense features after installing a language extension, open an issue in the repository of the language extension. 
+
+> **Tip:** For configuring and troubleshooting JavaScript IntelliSense see [here](/docs/languages/javascript#configuring-intellisense).
```

Comment:
For GitHub navigation to work include the .md extension.
use /docs/languages/javascript.md#configuring-intellisense.
We remove .md and add an underscore when processing the Markdown links.